metabrainz/templates/donations/donate.html

```diff
@@ -127,6 +127,10 @@ <h1 class="page-title">Make a Donation</h1>
       United States
     </p>
     <p>Please make checks payable to the <strong>MetaBrainz Foundation</strong>.</p>
+    
+    <h3>Flattr</h3>
+    <p>Follow this link to pay through Flattr:</p>
```

Comment:
Perhaps, "If you use Flattr to donate to your favourite projects, you can click below to Flattr MetaBrainz:". @mayhem or Christina may have a suggestion for even better wording.